.github/workflows/prepare-package-release.yml

```diff
@@ -3,13 +3,60 @@ on:
     workflow_dispatch:
         inputs:
             packages:
-                description: 'Enter a specific package to release, or releases separated by commas, ie @woocommerce/components,@woocommerce/number. Leaving this input blank will prepare to release all eligible packages.'
+                description: 'Enter a specific package to release, or packages separated by commas, ie @woocommerce/components,@woocommerce/number. Leaving this input to the default "-a" will prepare to release all eligible packages.'
                 required: false
                 default: '-a'
 jobs:
     prepare:
-        name: Run the prepare script
+        name: Run prepare script
         runs-on: ubuntu-20.04
         steps:
-            - name: Run
-              run: echo "hello world"
+            - uses: actions/checkout@v3
+            - uses: ./.github/actions/cache-deps
+              with:
+                  workflow_name: prepare-package-release
+                  workflow_cache: ${{ secrets.WORKFLOW_CACHE }}
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '7.4'
+
+            - name: Install PNPM
+              run: npm install -g pnpm@^6.24.2
+
+            - name: Install dependencies
+              run: pnpm install
+
+            - name: Execute script
+              run: ./tools/package-release/bin/dev prepare ${{ github.event.inputs.packages }}
+
+            - name: Print git status
+              run: git status
+
+            - name: Get current date
+              id: date
+              run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+            - name: Set all package string
+              id: all_description
+              if: ${{ github.event.inputs.packages == '-a'}}
+              run: echo "::set-output name=str::all packages"
+
+            - name: Set Specific packages string
```

Comment:
I've notice this step gets skipped even though I am not running `-a` as seen by this workflow run https://github.com/woocommerce/woocommerce/runs/7085703704?check_suite_focus=true I am not sure if that is expected?

Reply:
Yes, I wanted to differentiate between `-a` and something like `@woocommerce/components,@woocommerce/number` and tried doing a bash `if` statement with no luck. Instead I made two commands, this one and the one above it. One or the other should run, but not both.